docs/Changelog.md

```diff
@@ -8,6 +8,7 @@
   * openapi-circe-yaml: 0.6.0 -> 0.7.4 
 * [#5438](https://github.com/TouK/nussknacker/pull/5438) [#5495](https://github.com/TouK/nussknacker/pull/5495) Improvement in DeploymentManager API:
     * Alignment in the api of primary (deploy/cancel) actions and the experimental api of custom actions.
+* [#5783](https://github.com/TouK/nussknacker/pull/5783) Added allowedProcessingModes field to components endpoint
```

Comment:
let's show it from the user's black box perspective. If there is no such perspective and the change is e.g. technical, so maybe we should consider adding it to the migration guide.

Reply:
in this case it would be probably better not to mention this change anywhere. I think it doesn't fit to migration guide because this change doesn't affect current functionality in any way

Reply:
It affect code interfaces. Take a look at the summary of changes - there are changes in `components-api` module. It means, that authors of custom components will probably need to do modifications in their code.

From the user perspective, there also will be changes but they are hidden now - they will be available after the changes on the FE side. It would be nice to have links to all related PR's next to this entry but the author of FE side probably won't add the link to this PR.  Do you guys have idea how to handle it?

Reply:
assuming the REST Components API is the public API (not the internal one), we can say that from now on, we expose in some endpoint the information about processing modes. 

And @Dzuming please remember to add information in the changelog from the UI perspective, when your change is done.

Reply:
ok, I adapted changelog entry according to your suggestions, please verify

Reply:
it's ok for me. According to the @arkadius's comment, I guess we need to add a Migration Guide's entry with the changes in the components-api module

Reply:
Yes, please also add a MG entry

Reply:
Thanks. It's fine now for me. @mk-software-pl resolve the comment if it is fine for you as well